components/opc-ua/src/Microsoft.Azure.IIoT.OpcUa/src/Registry/Models/DiscoveryMode.cs

```diff
@@ -15,6 +15,11 @@ public enum DiscoveryMode {
         /// </summary>
         Off,
 
+        /// <summary>
```

Comment:
For consistency, please also add to end of the enum.   Even though technically this is only required for API models

Reply:
done